swim/test_utils.go

```diff
@@ -121,6 +122,44 @@ func bootstrapNodes(t *testing.T, testNodes ...*testNode) {
 		})
 		require.NoError(t, err, "node must bootstrap successfully")
 	}
+	return hostports
+}
+
+func waitForConvergence(t *testing.T, timeout time.Duration, testNodes ...*testNode) {
```

Comment:
Dear reviewers,

I've rewritten this method because it was causing flappy tests. Please have a look at the new version.